adbg/src/adbg_expect.c

```diff
@@ -195,9 +204,9 @@ bool Do_ADBG_ExpectComparePointer(
 	const char *const Value2Str_p
 	)
 {
-	if (!ADBG_AssertHelper(FileName_p, LineNumber, Result)) {
-		LOG_PRINTK(
-			"%s:%d: Expression \"%s %s %s\" (%p %s %p) is false",
+	if (!ADBG_AssertHelper(c, FileName_p, LineNumber, Result)) {
+		printk(
```

Comment:
Direct usage of printk is deprecated, it is better to use LOG_INF/_ERR/_DBG

Reply:
This was done similar to z_zassert implementation, which is also using printk under the hood. This was done because macro does formatting. Using LOG_* will make LOG_* formatting + xtest formatting. We want to minimize the difference between zephyr-optee-test and optee_test output which can be handy in future for example it will be easier to compare and parse output in CI (if someone dare make one). Also I'm not sure adbg internals is the right place for LOG_MODULE_REGISTER call.

Reply:
Also ADBG has it's own macro, called ADBG_LOG. I think the implementation of ADBG_Log using LOG_DBG, or replacing ADBG_LOG to LOG_DBG across all project is not needed.